### ISSUE TITLE
DBUS_SESSION_BUS_ADDRESS not set, dbus-launch question

### ISSUE THREAD

**Issue by cfebs** [tvrzna/emptty#26]
Hello! Been enjoying emptty, just had a question (maybe a bug report) about proper dbus launching.

My current version

```
➜ pacman -Qs emptty-git
local/emptty-git 0.3.0.r16.af974bf-1
    Dead simple CLI Display Manager on TTY
```

I noticed that recently `dunst` stopped working for chromium notifications and after researching that issue, found that my `$DBUS_SESSION_BUS_ADDRESS` env variable was not set.

I'm using emptty conf with just xinitrc script + `i3`

- emptty conf: https://github.com/cfebs/ansibles/blob/d425f80d2d4e4e6a07affd7cdeee98e3b558cc2e/roles/arch-i3-desktop/files/empttyconf
- xinitrc: https://github.com/cfebs/ansibles/blob/d425f80d2d4e4e6a07affd7cdeee98e3b558cc2e/roles/arch-i3-desktop/files/xinitrc

If I disable emptty, login via regular tty prompt, then use [`sx`](https://www.archlinux.org/packages/community/any/sx/) to start `X` + `i3`, my `$DBUS_SESSION_BUS_ADDRESS` is set correctly.

Should I be using `dbus-launch` or `DBUS_LAUNCH` emptty option in my init scripts? Or is this better left to systemd to start the daemon?

I'll continue trying to reproduce with the current non-git version: https://aur.archlinux.org/packages/emptty/ but haven't got there yet.

**Comment by cfebs** [comment]
Confirmed that installing latest non-git release `0.3.0-1` via https://aur.archlinux.org/packages/emptty/ does change behavior and "fixes" the issue.

My procedure there was 

```
paru -Rs emptty-git
paru -S emptty
# relink my config
reboot
# login and select i3
```

And then `$DBUS_SESSION_BUS_ADDRESS` exists in my environment!

**Comment by tvrzna** [comment]
Hello,
I'm glad you like this application and have your issue resolved already.

I would recommend to try, what helps you more, but I think `exec dbus-launch $@` in `.xinitrc` should do the trick.

**Comment by cfebs** [comment]
Thanks for the response!

Just to clarify, I'm seeing 2 different behaviors per version
- `0.3.0-1`: `DBUS_SESSION_BUS_ADDRESS` is set
- `0.3.0.r16.af974bf-1`: `DBUS_SESSION_BUS_ADDRESS` is not set

Is that a regression or is the new behavior intended?

**Comment by NXij** [comment]
I've got a missing env var on the latest aur git build too.

**Comment by tvrzna** [comment]
I've checked changes and found it, it is definitely a bug. I believe that @as3ii will update `emptty-git` package as soon as possible.

**Comment by as3ii** [comment]
updated `emptty-git` just now

**Comment by cfebs** [comment]
Fix confirmed with `emptty-git 0.3.0.r20.8f67308-1`!
